src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -239,6 +248,18 @@ typedef enum cpe_tags {
     CPE_MSU_NAME
 } cpe_tags;
 
+/**
+ * @brief Structure that contains information about
+ * a package or packages affected by an speficic CVE.
+ */
+typedef struct cve_vuln_pkg {
```

Comment:
Add a new field to indicate if this node is duplicated, which means that it was found in the NVD and the OVAL database.